bundles/org.openhab.binding.amberelectric/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="amberelectric"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="amber-electric">
+		<label>Amber Electric</label>
+		<description>Amber Electric - wholesale access to power prices</description>
+
+		<channels>
+			<channel id="elecprice" typeId="amber-electric-elecprice"/>
+			<channel id="elecstatus" typeId="amber-electric-elecstatus"/>
+			<channel id="clprice" typeId="amber-electric-clprice"/>
+			<channel id="clstatus" typeId="amber-electric-clstatus"/>
+			<channel id="feedinprice" typeId="amber-electric-feedinprice"/>
+			<channel id="feedinstatus" typeId="amber-electric-feedinstatus"/>
+			<channel id="nemtime" typeId="amber-electric-nemtime"/>
+			<channel id="spike" typeId="amber-electric-spike"/>
+			<channel id="renewables" typeId="amber-electric-renewables"/>
+		</channels>
+
+		<config-description>
+			<parameter name="apikey" type="text" required="true">
+				<label>API Key</label>
+				<description>API key from the Amber website</description>
+			</parameter>
+			<parameter name="nmi" type="text">
+				<label>NMI</label>
+				<description>NMI for your address (Optional)</description>
+			</parameter>
+			<parameter name="refresh" type="integer">
+				<label>Refresh Interval</label>
+				<description>Specifies the refresh interval in seconds.</description>
+				<default>60</default>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<channel-type id="amber-electric-elecprice">
+		<item-type>Number</item-type>
```

Comment:
For UoM support, this could be:
```suggestion
		<item-type>Number:EnergyPrice</item-type>
```

this will allow you to benefit from automatic currency conversions and calculations.

Reply:
Done! I looked for this UoM when first developing the binding ~6 months ago and couldn't  find it!